Mathlib/Order/SuccPred/CompleteLinearOrder.lean

```diff
@@ -0,0 +1,118 @@
+/-
+Copyright (c) 2023 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Order.SuccPred.Limit
+import Mathlib.Order.ConditionallyCompleteLattice.Basic
+
+/-!
+
+# Relation between `IsSuccLimit` and `iSup` in (conditionally) complete linear orders.
+
+-/
+
+open Order
+
+variable {ι α : Type*}
+
+/-! ### ConditionallyCompleteLinearOrder -/
```

Comment:
Can you either give a meaningful name to this subheading (not a Lean declaration name) or drop it? (that's why I was removing it in my previous comment)

Reply:
I thought this is common practice to put lean declaration names in subheadings? Anyway I can trop it if it is unnecessary. I just wanted a place to put the docstring about primed names.

Reply:
In subheadings? Not as a Lean declaration name, or at the very least in quotes, like
```
/-! ### `ConditionallyCompleteLinearOrder` -/
```

Reply:
Yeah maybe without the quotes. I removed the headings anyways